VimeoUpload/Upload/Descriptor System/Old Upload/OldUploadDescriptor.swift

```diff
@@ -47,30 +47,30 @@ public class OldUploadDescriptor: ProgressDescriptor, VideoDescriptor
     {
         didSet
         {
-            print("\(self.currentRequest.rawValue) \(self.identifier)")
+            print("\(self.currentRequest.rawValue) \(String(describing: self.identifier))")
```

Comment:
Here's another print statement that might be not be necessary? cc/@supperspidey @nicolelehrer